### PR TITLE
add cli promiscuous

### PR DESCRIPTION
**Pull request by mszczodrak**

### CODE REVIEW

File: src/cli/README.md [hunk 1]
```diff
@@ -613,6 +614,24 @@ Invalidate a prefix in the Network Data.
 Done
 ```
 
+### promiscuous start
```
Comment (jwhui) [inline src/cli/README.md]: nit: boolean properties generally are enable/disable

File: src/cli/README.md [hunk 2]
```diff
@@ -613,6 +614,24 @@ Invalidate a prefix in the Network Data.
 Done
 ```
```
Comment (jwhui) [inline src/cli/README.md]: Add documentation for retrieving the `promiscuous` property.

File: src/cli/cli.cpp [hunk 3]
```diff
@@ -1058,6 +1059,107 @@ void Interpreter::ProcessPollPeriod(int argc, char *argv[])
...
 
+void Interpreter::ProcessPromiscuous(int argc, char *argv[])
+{
+    ThreadError error = kThreadError_None;
+
+    if (argc == 0)
+    {
+        if (otIsLinkPromiscuous() && otPlatRadioGetPromiscuous())
+        {
+            sServer->OutputFormat("true\r\n");
```
Comment (jwhui) [inline src/cli/cli.cpp]: Would be better to have the status getter/setter match (e.g. `enable/disable` or `true/false`).

**Comment by jwhui** [comment]
LGTM, see my nits above.

**Comment by mszczodrak** [comment]
Thanks for the comments; changes applied.

**Comment by jwhui** [comment]
LGTM 👍

**Pull request merged**